pkgs/development/compilers/vyper/default.nix

```diff
@@ -10,18 +23,27 @@ let
     def __init__(foo: address):
         self.count = 1
   '';
-in
 
+in
 buildPythonPackage rec {
   pname = "vyper";
   version = "0.3.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7" || pythonAtLeast "3.10";
```

Comment:
I think it is better to move the `pythonAtLeast` to the `meta.broken` instead.

The reason for this is that I think this package will eventually build on Python 3.10 (at least, this is what I expect), while `disabled` is mostly used to mark this as "upstream doesn't give support".

Reply:
Ah, I see that some packages does this too. Forget it.

Reply:
This version doesn't support Python 3.10, that's why I set it on disabled. If this was updated to the current version on upstream (which indicates it supports Python 3.10), then would agree on moving it to `meta.broken`.

Let me try updating it to see if there are no issues.